examples/nodejs_postgresql/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    alpha.image.policy.openshift.io/resolve-names: '*'
+    app.openshift.io/vcs-ref: master
+    app.openshift.io/vcs-uri: https://github.com/pmacik/nodejs-rest-http-crud
+    deployment.kubernetes.io/revision: "2"
+    image.openshift.io/triggers: '[{"from":{"kind":"ImageStreamTag","name":"nodejs-rest-http-crud:latest"},"fieldPath":"spec.template.spec.containers[?(@.name==\"nodejs-rest-http-crud\")].image"}]'
+  labels:
+    app: nodejs-rest-http-crud
+    app.kubernetes.io/component: nodejs-rest-http-crud
+    app.kubernetes.io/instance: nodejs-rest-http-crud
+    app.kubernetes.io/name: nodejs
+    app.kubernetes.io/part-of: nodejs-rest-http-crud-app
+    app.openshift.io/runtime: nodejs
+    app.openshift.io/runtime-version: "1"
+  name: nodejs-rest-http-crud
+  namespace: service-binding-demo
```

Comment:
It would be really convenient to use this example to create a resource in a different namespace without having to remove the `metadata.namespace` field in the case the user can't create the `service-binding-demo`  by any reason.

What do you think about taking this as an opportunity to remove the `metadata.namespace` from all resources used in examples?

Reply:
@isutton Thanks for your suggestion, this is something that we've been postponing for a while, so yes this might be a good opportunity to that. However, this kind of change would affect most (if not all) of the other examples so I'd rather do the namespace change in a separate PR, just to keep the size of this at a reasonable level. Would you agree?

Reply:
Agreed; would you please create an issue to track this then?

Reply:
The tracking issue is created: #501